package-lock.json

```diff
@@ -1,8 +1,1843 @@
 {
   "name": "jx-docs",
   "version": "2.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
why this change?

Reply:
Didn't made any change in package-lock.json, just run `npm i` & it made a lot of changes